<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delService</class>
 <widget class="QDialog" name="delService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1199</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1199</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Car Workshop Management</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyświetl zlecenia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń zlecenie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyszukaj zlecenie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj zlecenie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>232</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Podaj ID zlecenia do usunięcia:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="theme1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1131</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>1051</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <zorder>picture</zorder>
  <zorder>theme1</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
  <zorder>tableView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
